packages/eslint-plugin-lit-a11y/README.md

```diff
@@ -64,6 +64,16 @@ In this case, you can configure `@apollo-elements/lit-apollo` as a valid lit-htm
 
 The reason for this check is that project may make use of multiple rendering libraries, which may have similar apis, like for example [`htm`](https://github.com/developit/htm), which also uses a `html` tagged template literal, but handles syntax differently.
```

Comment:
I made a typo just above here ^

> By default, only templates imported from 'lit-html' or 'lit-element' are exported.

That should be:

> By default, only templates imported from 'lit-html' or 'lit-element' are linted.

Could we take that along in this PR? 😬 sorry :p

Reply:
done